.github/workflows/nodejs-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Intacct JS SDK CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+  workflow_dispatch:
```

Comment:
for pull_request as empty, it means that this will trigger on each PR regardless of branch.

for workflow_dispatch, this allows us to trigger manual executions whenever we need to.